wp-content/themes/access/gulpfile.babel.js

```diff
@@ -343,19 +342,88 @@ gulp.task('images', callback => {
 /**
  * SVGs
  */
+let list = [];
+
+gulp.task('svgs:clean', (done) => {
+    del([
+      `${DIST}/svg/icons.*.svg`,
+    ]);
+
+    done();
+});
+
+gulp.task('svgs:list', () => gulp.src([
+    `${VIEWS}/**/*.twig`,
+    `${VIEWS}/**/*.vue`
+  ]).pipe(through.obj((chunk, encoding, callback) => {
+    const regex = /xlink:href="([\S]*)#([\S]+)"/g
+    let content = chunk.contents.toString('utf8');
+    let m;
+    while ((m = regex.exec(content)) !== null) {
+      list.push(m[2]);
+    }
+    callback(null, chunk);
+  }))
+);
 
-gulp.task('svgs', () =>
-  gulp.src(`${PATTERNS_ACCESS}/src/svg/*.svg`)
+gulp.task('svgs:add', () => gulp.src('assets/svg/icons.svg')
+  .pipe(through.obj((chunk, encoding, callback) => {
+     let iconsInUse = [
+      'icon-card-work-v2',
+      'icon-card-family-services-v2',
+      'icon-card-people-with-disabilities-v2',
+      'icon-card-cash-expenses-v2',
+      'icon-cash-expenses-v2',
+      'icon-card-food-v2',
+      'icon-card-housing-v2',
+      'icon-card-health-v2',
+      'icon-card-city-id-card-v2',
+      'icon-card-enrichment-v2',
+      'icon-card-education-v2',
+      'icon-card-child-care-v2',
+      'icon-child-care-v2',
+      'icon-city-id-card-v2',
+      'icon-education-v2',
+      'icon-enrichment-v2',
+      'icon-family-services-v2',
+      'icon-food-v2',
+      'icon-health-v2',
+      'icon-housing-v2',
+      'icon-people-with-disabilities-v2',
+      'icon-work-v2',
+      'icon-urgent'
```

Comment:
Are any of the other [alert icons](https://accesspatterns.cityofnewyork.us/icons#status-icons) needed? `icon-info` `icon-success` `icon-warning` I would add them to be safe.